pallets/ddc-payouts/src/mock.rs

```diff
@@ -315,7 +338,10 @@ impl<T: frame_system::Config> SortedListProvider<T::AccountId> for TestValidator
 }
 
 pub fn get_fees(cluster_id: &ClusterId) -> ClusterFeesParams {
-	if *cluster_id == FREE_CLUSTER_ID || *cluster_id == ONE_CLUSTER_ID {
+	if *cluster_id == FREE_CLUSTER_ID ||
+		*cluster_id == ONE_CLUSTER_ID ||
+		*cluster_id == CERE_CLUSTER_ID
+	{
```

Comment:
You can use `ReentrantMutex` to make a specific setup in a Visitor for a specific test. Take a look at an example in [ddc-staking mock.rs](https://github.com/Cerebellum-Network/blockchain-node/blob/deployment-0.9.30/pallets/ddc-staking/src/mock.rs#L260)